packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk

```diff
@@ -17,4 +17,4 @@ PKG_LONGDESC="audiodecoder.usf"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="kodi.audiodecoder"
-PKG_ADDON_PROJECTS="Generic Nvidia_Legacy RPi2 imx6 WeTek_Play"
+PKG_ADDON_PROJECTS="any"
```

Comment:
You should just drop this line. No other addon defines it.

Reply:
Thanks, applied.

Reply:
> You should just drop this line. No other addon defines it.

```
neil@nm-linux:~/projects/pullrequest_repos/LibreELEC.tv$ git grep PKG_ADDON_PROJECTS=
config/docker/package.mk:PKG_ADDON_PROJECTS="@PROJECTS@"
packages/addons/libraries/adafruit-libraries/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/addons/service/dispmanx_vnc/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/addons/service/slice/package.mk:PKG_ADDON_PROJECTS="Slice Slice3"
packages/addons/service/touchscreen/package.mk:PKG_ADDON_PROJECTS="Generic RPi"
packages/addons/tools/rpi-tools/package.mk:PKG_ADDON_PROJECTS="RPi"
packages/packages.mk.addon_template:# PKG_ADDON_PROJECTS="[project, only set when not any]"
```

If we really wanted to exclude RPi (the device and not the project) then setting `PKG_ADDON_PROJECTS="!RPi1"` would suffice.